src/startupscriptgenerator/node/scriptgenerator.go

```diff
@@ -135,6 +135,14 @@ func (gen *NodeStartupScriptGenerator) GenerateEntrypointScript() string {
 		startupCommand = common.ExtendPathForCommand(startupCommand, gen.SourcePath)
 	}
 
+	logger.LogInformation("Checking configuration to setup appinsight auto collection if needed")
+	scriptBuilder.WriteString("if [ -f ./" + common.ManifestFileName + " ]; then\n")
+	scriptBuilder.WriteString("    isInManifest=$(cat " + common.ManifestFileName + " | grep -c 'OryxInjectedAppInsight')\n")
```

Comment:
just `source` the manifest file (this was the reason we chose to use `key=value` format in the file) and access the variable name

Reply:
Thinking more we would be sourcing this file here as well as in line `62` in this file. I think we should do only once, maybe at the very beginning of the script